contracts/LiquidationsPeripheral.vy

```diff
@@ -243,7 +256,8 @@ def _computeLiquidationInterestAmount(principal: uint256, interestAmount: uint25
 @internal
 def _getNFTXVaultAddrFromCollateralAddr(_collateralAddress: address) -> address:
     assert self.nftxVaultFactoryAddress != empty(address), "nftx vault address not defined"
-    vaultAddrs: DynArray[address, 20] = INFTXVaultFactory(self.nftxVaultFactoryAddress).vaultsForAsset(_collateralAddress)
+    _unwrappedCollateralAddress: address = self._unwrappedCollateralAddressIfWrapped(_collateralAddress)
```

Comment:
Usually, we only used the `_` prefix for method arguments and not for method variables. For consistency's sake, should we keep the convention we use everywhere else?
```suggestion
    unwrappedCollateralAddress: address = self._unwrappedCollateralAddressIfWrapped(_collateralAddress)
```